src/data/python-quiz.ts

```diff
@@ -947,6 +947,33 @@ const pythonQuiz = [
     Explanation: "We cannot create a Python package without adding a __init__.py file, This file will tell Python to treat directories as modules.",
     Link: "https://www.freecodecamp.org/news/build-your-first-python-package/",
   },
+  {
+    Question: "What do partition() function do?",
+    Answer: "Splits the string at the first occurence and returns a tuple",
+    Distractor1: "Splits the string at the first occurence and returns a list",
+    Distractor2: "Splits the string equally",
+    Distractor3: "Split the string at any occurence of the given argument",
+    Explanation: "The partition() method searches for a specified string, and splits the string into a tuple containing three elements.",
+    Link: "https://www.w3schools.com/python/ref_string_partition.asp",
+  },
+  {
+    Question: "An empty/null statement in Python is",
+    Answer: "pass",
+    Distractor1: "continue",
+    Distractor2: "over",
+    Distractor3: "continue",
+    Explanation: "An empty statement is a statement that do nothing. Python offers pass statement as empty statement",
+    Link: "https://www.geeksforgeeks.org/how-to-write-an-empty-function-in-python-pass-statement/",
```

Comment:
Please use either a freeCodeCamp resource like an article or youtube video or use official python documentation.

The docs would be good in this case
https://docs.python.org/3/tutorial/controlflow.html#pass-statements